regenerate-website-2.sh

```diff
@@ -0,0 +1,1729 @@
+#!/bin/sh
+# regenerate-website - download & regenerate the Metamath website contents
+
+# Print the message $1 and exit with failure.
+fail () {
+  printf '%s\n' "$1" >&2
+  exit 1
+}
+
+# set -eu
+set -x
+
+if [ "$(whoami)" = 'root' ]; then
+    fail 'DO NOT run this as roo!! Execute /root/run-regenerate.sh instead!!'
```

Comment:
```suggestion
    fail 'DO NOT run this as root!! Execute /root/run-regenerate.sh instead!!'
```

Reply:
Maybe this was meant as pun ? One certainly should not act as root in a jumpy way.

Anyway, if this delays the merge, you can leave it like that.